File: 01_02_wsmt/project/client-go/models/author.go
package models

import "time"

type Author struct {
	ID        int64      `json:"ID,omitempty"`
	CreatedAt *time.Time `json:"CreatedAt,omitempty"`
	DeletedAt *time.Time `json:"DeletedAt,omitempty"`
	UpdatedAt *time.Time `json:"UpdatedAt,omitempty"`
	Name      string     `json:"name" xml:"name" yaml:"name" binding:"required"`
}

File: 01_02_wsmt/project/client-java/app/src/main/java/wsmt/rest/models/Author.java
package wsmt.rest.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(value = { "CreatedAt", "UpdatedAt", "DeletedAt" })
public class Author {
  @JsonProperty("ID")
  private int id;
  private String name;

  public int getId() {
    return id;
  }

  public void setId(int id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }
}
